dns.go

```diff
@@ -59,6 +65,7 @@ func (h *dnsHandler) ServeDNS(w dns.ResponseWriter, r *dns.Msg) {
 			},
 			A: address,
 		})
+		recordServed.WithLabelValues(strings.Split(domain, ".")[0]).Inc() // TODO clean this up
```

Comment:
probably want to remove the text of the `zone` from this, rather than this simple split